scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -135,12 +132,13 @@ else
 fi
 spaceID=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}")
 
-if [ "$central" == "true" ]; then
-    echo "looking for synctargets in the center"
-    kubectl ws "${kubectl_flags[@]}" "$espw"
+kubectl ws "${kubectl_flags[@]}" "$espw"
+if ! cluster_ns_name="$(kubectl "${kubectl_flags[@]}" -n kubestellar get configmap kbmap-"$spaceID" -ojsonpath="{.metadata.labels['kubestellar\.io\/kube-bind-id']}")"; then
+	echo "error reading kube-bind cluster namespace from the configmap"
```

Comment:
1. Write the error message to stderr rather than stdout.
2. Give the user some clue which ConfigMap is involved here.

Reply:
Addressed in #1338.